File: Java/LogicaProgramacaoJava(Eclipse)/exerciciosJava/src/exerciciosFaccat/Exercicio9.java
package exerciciosFaccat;

import java.util.Scanner;

/* 9) Escreva um algoritmo para ler o salário mensal atual de um funcionário e o percentual de reajuste.
Calcular e escrever o valor do novo salário */

public class Exercicio9 {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);

		System.out.println("Digite o salário mensal:");
		Float salarioMensal = sc.nextFloat();
		System.out.println("Digite o valor do percentual de reajuste:");
		Float percentualReajuste = sc.nextFloat();

		percentualReajuste = (salarioMensal * percentualReajuste) / 100;

		Float novoSalario = salarioMensal + percentualReajuste;

		System.out.println("O novo valor do salário com a taxa de reajuste é :" + novoSalario);
		
		sc.close();
	}
}

File: ExerciciosJavaScript(LógicadeProgramacao)/Exercicios 5 até o 39 Apostila Faccat/js/exercicio9.js
/* 9) Escreva um algoritmo para ler o salário mensal atual de um funcionário e o percentual de reajuste.
Calcular e escrever o valor do novo salário*/


alert("Algoritmo que calcula e escreve o valor do novo salário do funcionario com reajuste")

var salario = parseInt(prompt("Digite o valor do salário"))
var reajuste = parseInt(prompt("Digite o valor da porcentagem do reajuste"))

var reajusteP = (salario * reajuste) / 100
salario = reajusteP + salario

alert("O novo salário é: " + salario)


